ui-tests/src/test/java/org/openmrs/reference/VisitNoteTest.java

```diff
@@ -9,38 +9,41 @@
  */
 package org.openmrs.reference;
 
+import static org.junit.Assert.assertEquals;
+
 import org.junit.After;
 import org.junit.Before;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.openmrs.reference.groups.BuildTests;
-import org.openmrs.reference.page.*;
+import org.openmrs.reference.page.ActiveVisitsPage;
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.reference.page.EditVisitNotePage;
+import org.openmrs.reference.page.PatientVisitsDashboardPage;
+import org.openmrs.reference.page.VisitNotePage;
 import org.openmrs.uitestframework.test.TestData;
 import org.openmrs.uitestframework.test.TestData.PatientInfo;
 
-import static org.junit.Assert.assertEquals;
-
 public class VisitNoteTest extends LocationSensitiveApplicationTestBase {
 
-    private static final String DIAGNOSIS_PRIMARY = "CANCER";
-    private static final String DIAGNOSIS_SECONDARY = "MALARIA";
+    private static final String DIAGNOSIS_PRIMARY = "Gum Cancer";
+    private static final String DIAGNOSIS_SECONDARY = "Malaria";
     private static final String DIAGNOSIS_SECONDARY_UPDATED = "Pneumonia";
-
+    private PatientVisitsDashboardPage patientVisitsDashboardPage;
     private PatientInfo patient;
 
     @Before
     public void setup() {
         patient = createTestPatient();
         createTestVisit();
+        patientVisitsDashboardPage = new PatientVisitsDashboardPage(page);  
     }
 
     @Test
-    @Ignore //See RA-1223 for details
     @Category(BuildTests.class)
-    public void VisitNoteTest() throws Exception {
-
+    public void visitNoteTest() throws Exception {
 
+    	
```

Comment:
This is a minor concern though might be worth looking into! I would emit any redundant line after declaring a method. In short I would remove code line  `45`and `46 `for quality assurance.

Reply:
Remove the redundant code Line `LN-46`